fixtures/app/app.js

```diff
@@ -0,0 +1,4 @@
+import translations from './app.translations';
+import dictionary from '../dictionaries/en-US';
+
+console.log(translations, dictionary);
```

Comment:
Write something to the DOM. I usually do `npm start` without looking at the code first and thought the preview was dead.

'Check console' would have been sufficient